src/adapter/mysql.cr

```diff
@@ -5,6 +5,15 @@ require "mysql"
 class Granite::Adapter::Mysql < Granite::Adapter::Base
   QUOTING_CHAR = '`'
 
+  module Schema
+    TYPES = {
+      "AUTO_Int32" => "INT NOT NULL AUTO_INCREMENT PRIMARY KEY",
+      "AUTO_Int64" => "BIGINT NOT NULL AUTO_INCREMENT PRIMARY KEY",
```

Comment:
Can you add some quick tests for these in the adapter tests? I'd like to have them covered against accidental changes.

Reply:
Yes, I think it would be better to have a test. But I can not image a simple test. But, even now, Int32 and Int64 are used as primary keys in other specs, so I think the code has been covered. 😺